rustSlint/Dockerfile

```diff
@@ -107,6 +107,6 @@ ENV SLINT_BACKEND=winit-skia
 ENV SLINT_FULLSCREEN=1
 
 # ADD YOUR ARGUMENTS HERE
-CMD [ "./app" ]
+CMD [ "./__change__" ]
```

Comment:
I guess the environment variable in like 74 is unused then :)

Reply:
oh yeah, you are right, I'm rollback this change. Thanks for the review!